plugins/toolbox/src/components/Root/ToolsPage.tsx

```diff
@@ -274,7 +282,7 @@ export const ToolsPage = (props: ToolsPageProps) => {
               sx={{ justifyContent: 'space-between' }}
             >
               <InputBase
-                placeholder="Search"
+                placeholder={i18n_UNSAFE('toolsPage.input.search')}
```

Comment:
Is unsafe necessary?

Reply:
Leave for now until backstage fixes the types.